### DISCUSSION TITLE
Safe Area Blinking in  Resizable Sheet Until Transition Completes

### DISCUSSION THREAD

**Discussion by pmakris98** [SwiftfulThinking/SwiftfulRouting#76] [Q&A]
When presenting a resizable sheet using  showResizableSheet function, the safe area briefly blinks or flickers until the transition completes. This happens specifically when the sheet first appears, and it seems to be related to safe area adjustments during the presentation animation. Notably, when using the built-in .sheet view modifier, the transition works perfectly without any flicker. The custom sheet uses specific detents, and there may be state or layout recalculations involved causing this temporary blinking effect. Has anyone encountered this issue or found a way to stabilize the safe area during such transitions?  
Note: This bug seems to dissapear when the sheet content composed of a List.

**Comment by SwiftfulThinking** [comment]
This is fixed in Version 6.1! ".showResizableSheet" is now ".sheet(config:)"
